### ISSUE TITLE
How to submit bulk term request

### ISSUE THREAD

**Issue by ramonawalls** [obophenotype/bio-attribute-ontology#31]
Hey there. We are working on a bunch of new traits for the FuTRES project (https://futres.org/). Do you have a preferred method for us to do that? It looks like the OBO term genie is not working at the moment.

@megbalk

**Comment by cmungall** [comment]
if you could do PRs against the CSVs that would be super-duper

https://github.com/obophenotype/bio-attribute-ontology/tree/master/src/ontology/modules

This assumes you will need basic EQ compositional classes, maybe we should start by asking requirements. Do you need terms like this: https://github.com/futres/ovt/blob/master/src/entities/ovt_classes.csv

If you have difficulties then any way you like is fine, a bulk ticket with some kind of CSV/TSV structure should be fine

**Comment by ramonawalls** [comment]
I don't think we will actually need to specify that the traits are for vertebrates, although we probably have some classes for which we will want taxon constraints. Most of the traits should follow the EQ model. Some of them are a bit tricky, as they measure things like the length between two points on a bone. 

We may have a few requests for UBERON as well.

**Comment by cmungall** [comment]
We discussed this example on a POTATO call recently.

We have a number of anatomical lines in Uberon for things like this, e.g. line connecting inner canthi. These can then have simple length EQs (note that formally in BFO immaterial entities can't have qualities inhere in them but we are switching RO_0000052 to be more general).

Happy to add more lines. It should be easy to DPize so you would just need to fill in line in a TSV. But if the points are not defined this will need more detailed manual curation. Anyone should now be able to make PRs on Uberon, so this will all get easier

**Comment by ramonawalls** [comment]
Creating UBERON terms for lines is a great solution to these kinds of measurement traits.
